austation/code/game/objects/items/grenades/syndieminibomb.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/grenade/syndieminibomb
+  var/open_panel = 0
```

Comment:
Try to use boolean values with true/false checks

Reply:
this was just a copy paste of c4 wires

Reply:
be the change I want to see